content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -227,8 +227,17 @@ kubectl describe pvc <pvc_name>
 
 If the `PersistentVolumeClaim` has the status `FileSystemResizePending`, it is safe to recreate the pod using the PersistentVolumeClaim.
 
+FlexVolumes allow resize if the driver is set with the `RequiresFSResize` capability to true. 
```

Comment:
Can we please add `FlexVolume` to list of plugins that support resizing? It is above.

Reply:
Added to the list.

Reply:
done.